tests/test_kaggle_api_client.py

```diff
@@ -69,13 +70,13 @@ def test_download_corrupted_file_fail_integrity_check(self) -> None:
 
     @patch.dict("os.environ", {})
     def test_get_user_agent(self) -> None:
-        self.assertEqual(clients.get_user_agent(), "kagglehub/0.2.9")
+        self.assertEqual(clients.get_user_agent(), f"kagglehub/{kagglehub.__version__}")
```

Comment:
Nice! I'll change `kaggle-api` to use that, too.

Reply:
Thank you! Let me know if there is any other changes needed on the kagglehub side.